jenkinsapi_tests/unittests/test_crumbed_requester.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
```

Comment:
forgotten to remove the import after using it for debugging the tests below. I will remove it